lib/galaxy/util/user_agent.py

```diff
@@ -0,0 +1,34 @@
+import urllib
+
+import requests
+
+from galaxy.version import VERSION
+
+
+def __append_word_to_user_agent(word):
+
+    # set requests User-Agent
+    old_default_user_agent = requests.utils.default_user_agent
+
+    def new_default_user_agent(*args):
+        return f"{old_default_user_agent(*args)} {word}"
+
+    requests.utils.default_user_agent = new_default_user_agent
```

Comment:
We'll have to find another way to do this, we shouldn't do monkey patching on a third party library